content/case_histories/bookpurnong/survey.rst

```diff
@@ -3,3 +3,67 @@
 Survey
 ======
 
+Loop-loop system
+----------------
+
+DC resistivity can measure the ground conductivity (see Case History > Mt. Isa), but it requires direct connection between the instrument and the earth through cables and electrodes. Here we consider another type of measurement that relies on the inductive coupling between the instrument and the ground, a mechanism more suitable for portable or towed systems. Because it uses a loop as the transmitter and another loop as the receiver, it is often referred to as "loop-loop" system. 
```

Comment:
here you can use `` (see the :ref:`Mt. Isa Case History<mt_isa_index>)` ``to make a link (ie ``:ref:`Name on thing you want linked<link_anchor>`)``